admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js

```diff
@@ -149,7 +150,7 @@ export default function() {
       return this;
     },
     'unselectCombinations': function unselectCombinations() {
-      $('input.js-combination').prop('checked', false);
+      $('#toggle-all-combinations').prop('checked', false);
```

Comment:
Can you put a comment to explain that the bulk action check can check and uncheck all the checks.. :P